soda/scientific/tests/anomaly_detection/anomaly_detector_test.py

```diff
@@ -132,3 +152,176 @@ def test_anomaly_detector_evaluate(historical_measurements, historical_check_res
     assert diagnostic["value"] == expectation["value"]
     assert diagnostic["anomalyProbability"] == pytest.approx(expectation["anomalyProbability"])
     assert diagnostic["anomalyPredictedValue"] == pytest.approx(expectation["anomalyPredictedValue"])
+
+
+PROPHET_MODEL_PARAMS = {
+    "version": 1,
+    "request_params": {
+        "columns_mapping": {
+            "dataTime": "ds",
+            "value": "y",
+            "anomalyPredictedValue": "anomaly_predicted_value",
+            "anomalyProbability": "anomaly_probability",
+            "feedback": "feedback",
+        }
+    },
+    "feedback_processor_params": {
+        "output_columns": {
+            "ds": "ds",
+            "y": "y",
+            "delta": "external_regressor",
+            "skip_measurements": "skip_measurements",
+        }
+    },
+    "prophet_detector": {
+        "preprocess_params": {
+            "frequency": "D",
+            "warn_if_missing_values": True,
+            "fill_missing_dates": True,
+            "interpolation_kwargs": {"method": "linear"},
+            "assume_daily": True,
+        },
+        "anomaly_detection": {"n_points": 1, "criticality_threshold": 0.1},
+        "prophet_setup": {
+            "interval_width": 0.99,
+            "changepoint_range": 0.8,
+            "changepoint_prior_scale": 0.005,
+            "daily_seasonality": "auto",
+            "weekly_seasonality": "auto",
+            "yearly_seasonality": "auto",
+            "seasonality_mode": "multiplicative",
+            "growth": "linear",
+        },
+        "suppress_stan": True,
+    },
+    "response_params": {
+        "output_columns": {
+            "yhat": "anomaly_predicted_value",
+            "anomaly_probability": "anomaly_probability",
+            "warning_lower_than_or_equal": "warning_lower_than_or_equal",
+            "warning_greater_than_or_equal": "warning_greater_than_or_equal",
+            "critical_lower_than_or_equal": "critical_lower_than_or_equal",
+            "critical_greater_than_or_equal": "critical_greater_than_or_equal",
+        }
+    },
+}
+
+
+@pytest.mark.parametrize(
```

Comment:
I would create an anomaly_detection_assets.py folder to get the static dictionaries from there to make it more organized.

Reply:
Yes makes sense. 👍  Shall we do that when we start refactoring/creating unit tests for anomaly detection? I.e. this ticket: https://sodadata.atlassian.net/browse/CLOUD-2804